schema/src/ProjectInfo.yaml

```diff
@@ -0,0 +1,227 @@
+id: https://framework.regen.network/schema/ProjectInfo
```

Comment:
Do we want to remove the `id` field here? I think this is something @clevinson and I meant to do per a conversation with @aaronc. The id of a schema should be it's iri - we can use the Name to indicate the type of schema

Reply:
I don't understand the motivation to migrate schema ids to our hash IRIs. It seems useful to keep them semantic and human readable.

Reply:
looks like a schema id is required by some of the linkml commands